# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: architecture/riscv/package/*/hotfix-spinal.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-spinal-v5.11/drivers/net/ethernet/spinal/spinal_lib_mac.c.vanilla	2021-04-30 15:56:32.909126707 +0200
+++ linux-spinal-v5.11/drivers/net/ethernet/spinal/spinal_lib_mac.c	2021-04-30 15:56:40.964127122 +0200
@@ -480,4 +480,4 @@
 module_platform_driver(spinal_lib_mac_driver);
 
 MODULE_AUTHOR("Charles Papon");
-MODULE_LICENSE("Dual GPL");
+MODULE_LICENSE("GPL");
